Source/WebCore/platform/ios/PasteboardIOS.mm

```diff
@@ -187,52 +183,45 @@ static int64_t changeCountForPasteboard(const String& pasteboardName = { }, cons
     // plain text for this abstract UTI. In almost all cases, the more correct choice would be to write to
     // one of the concrete "public.plain-text" representations (e.g. kUTTypeUTF8PlainText). In the future, we
     // should consider removing support for reading plain text from "public.text".
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    text.text = strategy.readStringFromPasteboard(itemIndexToQuery, kUTTypePlainText, m_pasteboardName, context());
+    text.text = strategy.readStringFromPasteboard(itemIndexToQuery, String(UTTypePlainText.identifier), m_pasteboardName, context());
```

Comment:
No need for `String()` here.

Reply:
Would be nice to address prior to landing.

Reply:
Sorry  I missed that -- thank you for catching it. I'll do a double-check before landing.